.github/workflows/test.yml

```
@@ -67,95 +67,34 @@ jobs:
-        if: matrix.os == 'depot-windows-2022-8'
-        uses: KyleMayes/install-llvm-action@v2.0.8
-        with:
-          version: "21"
-
-      - name: build (depot-windows-2022-8)
-        if: matrix.os == 'depot-windows-2022-8'
-        run: |-
-          vcpkg install libsodium
-          $env:SODIUM_LIB_DIR="C:\vcpkg\packages\libsodium_x64-windows\lib"
+      - name: "Common Test Actions"
+        uses: ./.github/actions/common-test
```

actionlint (1.7.9)

description is required in metadata of "Common Test Running Actions" action at "/home/jailuser/git/.github/actions/common-test/action.yml"

(action)